Source/Creation/Frame/Content/ID3GroupingFrameCreator.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ID3GroupingFrameCreator.swift
+//
+//  Created by Fabrizio Duroni on 26/02/2018.
+//  2018 Fabrizio Duroni.
+//
+
+import Foundation
+
+class ID3GroupingFrameCreator: ID3StringFrameCreator {
```

Comment:
Rename this to `ID3iTunesGroupingFrameCreator` because this is not a standard mp3 frame but an iTunes specific one.